recipes/current.rb

```diff
@@ -20,7 +20,7 @@
 
 apt_repository 'chef-current' do
   uri 'https://packagecloud.io/chef/current/ubuntu/'
-  key node['apt-chef']['key']
+  key node['apt-chef']['gpg']
```

Comment:
I'm not sure if I prefer `key` because thats the property on this resource, or `gpg` because its more clear about its intent.